Tests/UnitTests/Core/Surfaces/AnnulusBoundsTests.cpp

```diff
@@ -12,14 +12,14 @@
 #include <boost/test/output_test_stream.hpp>
 // clang-format on
 
-#include <limits>
-#include <fstream>
-
+#include "Acts/Definitions/Algebra.hpp"
 #include "Acts/Surfaces/AnnulusBounds.hpp"
 #include "Acts/Tests/CommonHelpers/FloatComparisons.hpp"
-#include "Acts/Definitions/Algebra.hpp"
 #include "Acts/Utilities/Helpers.hpp"
 
+#include <fstream>
+#include <limits>
+
```

Comment:
Does this need to be part of the changeset?

Reply:
I think this another formatting change. I should go back to using the docker image of clang-format.